### PR TITLE
Separate api class

### PR DESCRIPTION
**Pull request by debsmita1**
GetApi.ts file contains exported functions which returns api-s as strings. So when API call is to be made, importing and calling the function would serve the purpose.

### CODE REVIEW

File: src/actions/Sidebar.ts [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 
 import { Dispatch } from "redux";
 
 import { IStateSidebar } from '../reducers/Sidebar';
 
 import IWorkspace from '../models/Workspace';
 
+import {workspacesApi} from '../apicalls/GetApi';
+
+import {workspaceApi} from '../apicalls/GetApi';
```
Comment (waveywaves) [inline src/actions/Sidebar.ts]: Have you called the same file twice ?

Comment (waveywaves) [inline src/actions/Sidebar.ts]: Have you imported the same file twice ?

Comment (debsmita1) [reply src/actions/Sidebar.ts]: No, one is workspace-s-Api and the other one is workspace-Api

Comment (waveywaves) [reply src/actions/Sidebar.ts]: Can you import it like, `import {workspacesApi, workspaceApi} from ...`

Comment (debsmita1) [reply src/actions/Sidebar.ts]: Changes Done.

**Pull request merged**